proof/refine/RISCV64/Invariants_H.thy

```diff
@@ -842,19 +1178,42 @@ definition sch_act_sane :: "kernel_state \<Rightarrow> bool" where
 abbreviation
   "sch_act_not t \<equiv> \<lambda>s. ksSchedulerAction s \<noteq> SwitchToThread t"
 
-definition idle_tcb'_2 :: "Structures_H.thread_state \<times> machine_word option \<Rightarrow> bool" where
-  "idle_tcb'_2 \<equiv> \<lambda>(st, ntfn_opt). (idle' st \<and> ntfn_opt = None)"
+definition
+  idle_tcb'_2 :: "Structures_H.thread_state \<times> obj_ref option \<times> obj_ref option \<times> obj_ref option
```

Comment:
name should go on same line as `definition`, type can be on next line if it is too long.

(the idea is that you can grep for "definition idle_tbc" and find the def)